### ISSUE TITLE
match not reading vcf correctly

### ISSUE THREAD

**Issue by stubbsrl** [amwenger/svpack#8]
I wasn't getting an output looking for matches in two VCFs, so I  tested match by comparing a VCF against itself. 

$SVP/svpack match groupA.vcf.gz groupA.vcf.gz > groupA.match.test.vcf

This produced a VCF with only the heading and no matches. There aren't any error messages. The svpack functions filter and consequence both run correctly my vcf files. The VCFs were created with LUMPY via smoove. 

Here is one line with a few samples from my VCF as I think the error must have to do formatting or a mismatch in what the script is searching for and what is in my vcf. Any idea where the problem is?

chrom_01	69902	27466	N	<DUP>	68.33	.	SVTYPE=DUP;SVLEN=16531;END=86433;STRANDS=-+:6;IMPRECISE;CIPOS=-489,29;CIEND=-30,664;CIPOS95=-94,9;CIEND95=-9,113;SU=6;PE=6;SR=0;SNAME=P008_WF12:5;ALG=PROD;GCF=0.32803;AN=80;AC=2	GT:GQ:SQ:GL:DP:RO:AO:QR:QA:RS:AS:ASC:RP:AP:AB:DHFC:DHFFC:DHBFC	0/0:28:0.01:-0,-3,-5:32:32:0:31:0:22:0:0:9:0:0:0.9375:0.882353:0.882353	0/0:29:0:-0,-3,-6:33:33:0:32:0:20:0:0:12:0:0:0.952381:0.909091:0.909091	0/0:26:0.01:-0,-3,-5:30:30:0:29:0:20:0:0:9:0:0:0.933333:1.16667:0.933333

**Comment by amwenger** [comment]
It looks like the ALT column may be missing:

```
#CHROM	chrom_01
POS	69902
ID	27466
REF	N
ALT	68.33
QUAL	.
FILTER	SVTYPE=DUP;SVLEN=16531;END=86433;STRANDS=-+:6;IMPRECISE;CIPOS=-489,29;CIEND=-30,664;CIPOS95=-94,9;CIEND95=-9,113;SU=6;PE=6;SR=0;SNAME=P008_WF12:5;ALG=PROD;GCF=0.32803;AN=80;AC=2
INFO	GT:GQ:SQ:GL:DP:RO:AO:QR:QA:RS:AS:ASC:RP:AP:AB:DHFC:DHFFC:DHBFC
FORMAT	0/0:28:0.01:-0,-3,-5:32:32:0:31:0:22:0:0:9:0:0:0.9375:0.882353:0.882353
```

Adding the ALT allele of `<DUP>` between `N` and `68.33` makes it work for me.  See 
* [add_alt.txt](https://github.com/PacificBiosciences/svpack/files/7481133/add_all.txt) - works
* [no_alt.txt](https://github.com/PacificBiosciences/svpack/files/7481135/no_alt.txt) - does not work

**Comment by stubbsrl** [comment]
Thanks for the response. The attached file has the ALT column and produces the same result. I piped out the first 1000 lines of my VCF (including the very long header) and attached it. I looked into the code and my files have "SVTYPE" in the info field so it must be in the actual match step. 

This reduced file produces the same output as the complete file and I am able to successfully run svpack filter on the attached reduced vcf.

I run 
svpack match 1000linessample.vcf.gz 1000linessample.vcf.gz 
and get just the header as output.
